File: php/class.php
<?php
// PHP class example
class Person {
    private $name;
    private $age;

    public function __construct($name, $age) {
        $this->name = $name;
        $this->age = $age;
    }

    public function greet() {
        return "Hello, my name is {$this->name} and I'm {$this->age} years old.";
    }

    public function birthday() {
        $this->age++;
        return "Happy Birthday! You are now {$this->age} years old.";
    }

    public function capitalizeName() {
        $this->name = ucfirst($this->name);
        return "Name capitalized: {$this->name}";
    }
}

$person = new Person("bob", 25);
echo $person->greet() . "\n";
echo $person->birthday() . "\n";
echo $person->capitalizeName() . "\n";
?>

File: python/class.py
# Python class example
class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age
    
    def greet(self):
        return f"Hello, my name is {self.name} and I'm {self.age} years old."

    def birthday(self):
        self.age += 1
        return f"Happy Birthday! You are now {self.age} years old."

if __name__ == "__main__":
    person = Person("Alice", 30)
    print(person.greet())
    print(person.birthday())
